.gitignore

```diff
@@ -16,6 +16,7 @@
 .env.development.local
 .env.test.local
 .env.production.local
+.idea
```

Comment:
.idea to katalog mojego edytora. Dodałam go tutaj żeby przyadkiem nie dodać plików, które nie powinny tu być